src/js/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "react-dom";
+import { Provider } from "react-redux";
+import store from "./store/index";
```

Comment:
@nvtcp9x Fixed the codes you mentioned above.

I wonder if I should also resolve the paths from like `src/js/reducers` `src/js/actions` `src/js/constants` `src/js/store`. 

In real project, are these paths resolved like `import store from "store/index"` ?

Reply:
It's not necessary because we just need resolve which path that we use many times in our projects